src/_tbds.scss

```diff
@@ -0,0 +1 @@
+@import "index";
```

Comment:
This file effectively duplicates what `index.scss` is doing (imports the system), and I think we should combine them. This file is also a Sass manifest, not a partial, and so it should not have an underscore prefix in the filename.

So, we should…

1. Delete this file.
1. Rename `src/index.scss` to `src/tbds.scss`
1. Update the path in the [installation instructions in the readme](https://github.com/thoughtbot/design-system#installation-for-ruby-on-rails):
    ```diff
    -@import "@thoughtbot/design-system/src/index";
    +@import "@thoughtbot/design-system/src/tbds";
    ```